### ISSUE TITLE
about result

### ISSUE THREAD

**Issue by ccatian** [xujian000/DAF-Net#4]
作者您好，在复现您的这篇文章的时候，训练损失好像一直没有变化，
<img width="300" alt="训练" src="https://github.com/user-attachments/assets/ccbffde1-98fc-4997-afa9-3a749e4d11b7">
此外得到的结果有很多类似于条纹状的噪声
<img width="660" alt="结果" src="https://github.com/user-attachments/assets/13c1e249-d7c3-4722-9ef7-d259898aa0b0">，想请问下是怎么回事呢？具体怎么解决？

**Comment by xujian000** [comment]
Hi, The logs for the code running normally should look like this. This is the result I just ran. Have you made significant changes? If changes to the code are necessary, please modify it step by step from the initial state and check the status at each step.
![28ce4403-5769-43fc-acb0-c6df61d41d28](https://github.com/user-attachments/assets/87202acd-692a-47c0-92c3-e9232a71e5ca)

**Comment by ccatian** [comment]
Yes, I modified the display part of the code, but the main parts such as the network and its loss function have not been changed. I will check again carefully. Thank you very much for your reply and help.
